src/libs/maintaining-std.md

```diff
@@ -223,8 +223,13 @@ Features can be stabilized in a PR that replaces `#[unstable]` attributes with `
 
 You can find the right version to use in the `#[stable]` attribute by checking the [Forge].
 
+### When a `const` function is being stabilized
```

Comment:
If it is an intrinsic being exposed that is const-stabilized then the language team should also be pinged.

Reply:
Also btw, there's some notes here: https://github.com/rust-lang/rustc-guide/blob/master/src/stability.md#allow_internal_unstable